### PR TITLE
Add quantized::softmax to fc list

### PR DESCRIPTION
**Pull request by janeyx99**
Hoping to fix regression from https://hud.pytorch.org/minihud#1bcae0d10e1c4eddf07f9e60ced9b4f3c2c04b1f

Adding quantized::softmax to list until 4/15/22.

### CODE REVIEW

**Review by albanD — APPROVED** [review]
LGTM
